services/API-service/src/api/event/dto/event-place-code.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { IsNotEmpty, IsNumber } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class EventPlaceCodeDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsNumber()
+  public eventPlacecodeId: number;
```

Comment:
`eventPlacecodeId` to `eventPlaceCode`

unless there is a difference between the `Place Code Id` and `Place Code` in which case use `eventPlaceCodeId` instead of `eventPlacecodeId`